configure

```diff
@@ -718,25 +720,11 @@ def configure_v8(o):
 def configure_openssl(o):
   o['variables']['node_use_openssl'] = b(not options.without_ssl)
   o['variables']['node_shared_openssl'] = b(options.shared_openssl)
-  o['variables']['openssl_no_asm'] = (
-    1 if options.openssl_no_asm else 0)
+  o['variables']['openssl_no_asm'] = 1 if options.openssl_no_asm else 0
```

Comment:
It's not a boolean, its string true or false but the openssl build system requires 1 or 0.